### DISCUSSION TITLE
CARLA data open for sales!

### DISCUSSION THREAD

**Discussion by yqlin9512** [carla-simulator/carla#6714] [Show and tell]
we build some terrain in China\Europe cities and can be annotated. Also build the scenarios for ADAS/AD sumulation test\CIDAS\SOTIF. Now open for trial and can be sales!
![图片1](https://github.com/carla-simulator/carla/assets/63838086/07dbbc4f-2de0-446b-ad76-17837eda3033)

**Comment by AAT24** [comment]
Is this still available?

**Reply by iae-yqlin** [reply]
Yes, it is available in the long-term.
You can contact "info@iae-tech.com" with your info and requirement. 
We are looking forward to your message.
